_glaredb/integrations/hyperquery.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Hyperquery
+parent: Integrations
+---
+
+# Hyperquery
+
+[Hyperquery] is a data notebook built for speed, visibility, and collaboration.
+You can write and execute SQL queries or Python cells directly within a WYSIWYG
+(what you see is what you get) notebook, and augment this work with rich
+formatting options.
+
+In this guide, we'll walk through how to get starting with GlareDB in
+Hyperquery.
+
+## Running GlareDB in Hyperquery
+
+To get started, first either select an existing notebook, or create a new.
+To create a new notebook, click **Create a new page**.
+
+![create]
+
+And now we have a new notebook ready to go. To install GlareDB in the notebook
+type `/`, then select **Python block**.
+
+![python block]
+
+This will place a new block for executing python in the notebook. We'll use this
+first block to install the [glaredb python library]. To do so, execute the
+following command in the code block:
+
+```text
+!pip install glaredb
+```
+
+If everything installed succesffuly, you should be able to import and use
+glaredb in a new Python block:
+
+```python
+import glaredb
+
+con = glaredb.connect()
+con.sql("select 'hello from hyperquery").to_pandas()
+```
+
+And you should see the follwing in your notebook:
+
+![success]
+
+## Connecting to GlareDB Cloud from Hyperquery
+
+Connecting to [GlareDB Cloud] to your deployment is easy. If you're a new user,
+a deployment is automatically created for you. If you have multiple, select the
```

Comment:
I've often found it gauche to declare something is easy in documentation: what if the user doesn't find it easy? what if something's broken and it doesn't work? Safe to omit. 

----

a new (glaredb) user not a new Hyperquery user?

multiple [deployments] ->